tests/07-remove.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+test_description='Test the remove command'
+
+WHEREAMI=$(dirname "$(realpath "$0")")
+. $WHEREAMI/setup.sh
+
+test_expect_success 'cabin remove tbb mydep toml11' '
+    mkdir remove_test &&
+    cp "$WHEREAMI/../cabin.toml" remove_test/cabin.toml &&
+
+    test_when_finished "rm -rf remove_test" &&
+
+    (
+        cd remove_test &&
+        "$CABIN_BIN" remove tbb mydep toml11 >../actual 2>&1
```

Comment:
I think we only need stderr, not stdout.

Reply:
There is a warning and a info both for this command, so we also need the stdout

Reply:
They both are from Logger, right? All logger outputs go into stderr.

https://github.com/cabinpkg/cabin/blob/51c69e3cd643efc68145771f56ac581ac100e86a/src/Logger.hpp#L180